website/src/routes/index.svelte

```diff
@@ -1,28 +1,57 @@
 <script>
-  import { onMount } from "svelte";
-
-  let slideIndex = 0;
+  import {onMount, afterUpdate, onDestroy} from 'svelte';
+  import {firestore} from '../firebase';
+  let photos = [];
+
+  let slidesTimeout;
+
+  function fetchHomePhotos() {
+    return new Promise((resolve, reject) => {
+      firestore
+        .collection('home')
```

Comment:
You can retrieve the document directly like this `firestore.doc('home/home')` that way you don't need to capture the first one.